providers/vsphere/resources/vsphere.lr

```diff
@@ -132,6 +132,10 @@ private vsphere.datacenter @defaults("moid name") {
   vms() []vsphere.vm
   // Clusters in the datacenter
   clusters() []vsphere.cluster
+  // Distributed virtual switches
+  distributedSwitches() []vsphere.vswitch.dvs
+  // List of distributed virtual port groups
+  distributedPortgroups() []vsphere.vswitch.portgroup
```

Comment:
maybe `distributedPortGroups`

Reply:
Agreed.
VMware uses separate words for it:
>  A distributed port group ...

Reply:
@mm-weber FYI
You might need to change this also in the policy.